app/controllers/last_seen_issues_controller.rb

```diff
@@ -0,0 +1,31 @@
+class LastSeenIssuesController < ApplicationController
+
+    include QueriesHelper
+    helper :all
+
+    def open_popup
+        return render :action => 'open_popup'
+    end
+
+    def show_last_seen_issues 
+        @issues = []
+        @remove_ids = []
+        query = retrieve_query_from_session(IssueQuery)
+        @columns = query.nil? ? retrieve_query(IssueQuery).inline_columns : query.inline_columns
+        ids = params[:ids].split(",");
```

Comment:
Ok, you made the check on javascript, but we should also make the check on server. We don't want the server to crash on misbehaving client (since this is a public api called by any js code)

Reply:
Done.